### ISSUE TITLE
512x512 colab demo

### ISSUE THREAD

**Issue by tjess78** [AliaksandrSiarohin/first-order-model#135]
Hi,
thanks for your great repo!! Could you please make a second colab Demo for 512x512 material?? I've tried it myself but failed on my lack of knowledge.
greetings
tjess78

**Comment by AliaksandrSiarohin** [comment]
It is not supposed to be used in 512. You can try https://github.com/tg-bomze/Face-Image-Motion-Model they use external super-resolution method.

**Comment by bigboss97** [comment]
> 
> 
> It is not supposed to be used in 512. You can try https://github.com/tg-bomze/Face-Image-Motion-Model they use external super-resolution method.

I tried on that. It crashed at "Transform image" and said: ran out of RAM  :-(

**Comment by ayush9198gupta** [comment]
Yes same issue , I have tried with higher resolution image but Colab session ran out of RAM.

and I am using Pro Version(Colab)

**Comment by bigboss97** [comment]
Even just 256x256, it crashed. Maybe my video was too long. Luckily, 256res works on my local machine.

**Comment by bigboss97** [comment]
I tried with a 10s video and it worked. But it failed at download. When I used "Save Video As..." I still got 256x256 only. I doubt it that it can make 512x512
![Screenshot - 20-07-10 14 40 43](https://user-images.githubusercontent.com/34834507/87117439-e717f000-c2bb-11ea-90da-7df673f976e3.png)

**Comment by bigboss97** [comment]
I used a 512 (25s) video and also managed to get to the last step "Download final videos". Unfortunately the video is still 256  :-(
I skipped "Frames interpolation" and went to "Upscale Resolution". But ended up with:
![Screenshot - 20-07-10 16 22 28](https://user-images.githubusercontent.com/34834507/87123924-0a966700-c2cb-11ea-890d-3d5bb3ed68d7.png)
